build.gradle

```diff
@@ -372,9 +385,38 @@ task createApiDocsRedirect {
   }
 }
 
+task publishDocs {
+  dependsOn 'closeAndReleaseRepository'
+  doLast {
```

Comment:
What do you think about adding an additional task here to pull the latest gh-pages? Or possibly this should be done at doc generation time. Otherwise if the gh-pages is not at head, the git push will fail.